ingenialink/canopen/network.py

```diff
@@ -1076,10 +1082,11 @@ def _get_available_kvaser_devices() -> List[Dict[str, Any]]:
             if "Virtual" not in can.interfaces.kvaser.get_channel_info(channel)
         ]
 
-    @staticmethod
-    def _reload_kvaser_lib() -> None:
-        """Reload the Kvaser library to refresh the connected transceivers."""
-        canInitializeLibrary = get_canlib_function("canInitializeLibrary")
-        canUnLoadLibrary = get_canlib_function("canUnloadLibrary")
-        canUnLoadLibrary()
-        canInitializeLibrary()
+    def _reload_kvaser_lib(self) -> None:
+        """Reload the Kvaser library to refresh the connected transceivers.
+        Only when no drives are connected"""
+        if self.__device == CAN_DEVICE.KVASER.value and not self.servos:
```

Comment:
This function now is deceiving. I would have placed this condition on _get_available_kvaser_devices, before calling reload